### ISSUE TITLE
Can we go back to the old notification system?

### ISSUE THREAD

**Issue by Tiffys** [inbasic/ignotifier#647]
I really don't like the notifier now using the Windows notification center instead of the white pop up card with gmail logo it used to use.

Some way to go back to the old one would be nice.

**Comment by Avrution** [comment]
Agreed, there should at least be an option.  The notifications don't even work for me in this newest version.

**Comment by Avrution** [comment]
Posted this in another thread, but this will get you close to how things were if you happen to use firefox - not sure about Chrome

Open about:config
Search for "alerts.useSystemBackend"
Change to "false"
